ros2node/ros2node/api/__init__.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from ros2cli.node.strategy import NodeStrategy
+
+# TODO(mikaelarguedas) revisit this once it's specified
+HIDDEN_NODE_PREFIX = '_'
```

Comment:
Both usages in this patch are kind of unrelated. Therefore I didn't introduce a dependency for this. This constant should be defined in a lower level package like `rclpy` (hence the TODO).